components/camel-zendesk/pom.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
```

Comment:
The license header could be formatter nicer. Maybe copy a header from one of the existing components that look good